README.md

```diff
@@ -63,22 +65,24 @@ ___
 import { NativeModules } from 'react-native';
 
 //Assign our module from NativeModules and assign it to a variable
-var SpotifyAuth = NativeModules.SpotifyAuth;
+var SpotifyModule = NativeModules.SpotifyModule;
```

Comment:
I don't believe this is correct, cannot import `SpotifyModule`, file is still called `SpotifyAuth`